rfd/0073-idp-initiated-login.md

```diff
@@ -0,0 +1,67 @@
+---
+authors: Joel Wejdenstal <jwejdenstal@goteleport.com>
+state: draft
+---
+
+# RFD 73 - RFD 73 - IdP-initated Login Flows for SAML and OIDC
+
```

Comment:
@xacrimon please add the reviewers section.

Please include @reedloden as a required approver from the security team side as this has a potential security impact. @reedloden can you please review possible attack vectors and mitigations. Are missing anything?

Reply:
Added